contrib/terraform/vsphere/variables.tf

```diff
@@ -0,0 +1,96 @@
+## Global ##
+
+variable prefix {
+  default = ""
+}
+
+variable "machines" {
+  description = "Cluster machines"
+  type = map(object({
+    node_type = string
+    ip      = string
+  }))
+}
+
+variable inventory_file {
+  default = "inventory.ini"
+}
+
+variable "network" {
+  default = "VM Network"
+}
+
+variable "gateway" {
+  default = ""
```

Comment:
Since this default value cannot actually be used, I would rather not set it, so that Terraform asks for it.